packages/cactus-plugin-ledger-connector-corda/src/main/typescript/api-client/corda-api-client.ts

```diff
@@ -0,0 +1,152 @@
+import { Observable, ReplaySubject } from "rxjs";
+import { finalize, share } from "rxjs/operators";
+import {
+  Logger,
+  LogLevelDesc,
+  LoggerProvider,
+  Checks,
+} from "@hyperledger/cactus-common";
+import { ISocketApiClient } from "@hyperledger/cactus-core-api";
+import {
+  DefaultApi,
+  GetMonitorTransactionsV1ResponseTx,
+} from "../generated/openapi/typescript-axios";
+import { Configuration } from "../generated/openapi/typescript-axios/configuration";
+
+const DEFAULT_POOL_RATE_MS = 5000;
```

Comment:
@outSH Is it pooling or polling? Or is it both, e.g. polling the pool?

Reply:
@petermetz Fixed this and couple other misspells :)